crates/sui-core/src/authority/epoch_start_configuration.rs

```diff
@@ -126,6 +132,18 @@ pub struct EpochStartConfigurationV5 {
     coin_deny_list_obj_initial_shared_version: Option<SequenceNumber>,
 }
 
+#[derive(Serialize, Deserialize, Debug, Eq, PartialEq)]
+pub struct EpochStartConfigurationV6 {
+    system_state: EpochStartSystemState,
+    epoch_digest: CheckpointDigest,
+    flags: Vec<EpochFlag>,
+    /// Do the state objects exist at the beginning of the epoch?
+    authenticator_obj_initial_shared_version: Option<SequenceNumber>,
+    randomness_obj_initial_shared_version: Option<SequenceNumber>,
+    coin_deny_list_obj_initial_shared_version: Option<SequenceNumber>,
+    bridge_obj_initial_shared_version: Option<SequenceNumber>,
```

Comment:
What is this used for?

Reply:
`bridge_exists` function, which in turn will control whether the bridge is created during the next reconfiguration.